content/docs/how-we-work/baseline/_index.md

```diff
@@ -11,6 +11,8 @@ Adopting the baseline on a project enhances operational efficiency. It ensures q
 
 At this time, this baseline is standardized for our projects in [CDM](https://www.axelerant.com/delivery-engagement-models/continuous-development-and-maintenance) but will eventually also apply to our [managed projects](https://www.axelerant.com/delivery-engagement-models/managed-projects).
 
+To support the adoption of the baseline, baseline templates are available in our private GitHub repository: [Baseline Templates Repository](https://github.com/axelerant/baseline-templates/tree/main). These templates provide essential configurations and starting points for projects, helping teams ensure that baseline requirements are met with minimal effort.
```

Comment:
@Yemaneberhan-Lemma, please point them to the IDP as @zeshanziya mentioned. There is still a gap here as we saw @Kalaiselvan88 read this document and could not find the information about the IDP.